package.json

```diff
@@ -21,14 +21,15 @@
   },
   "devDependencies": {
     "@netlify/plugin-nextjs": "^5.8.1",
+    "@stately-cloud/schema": "^0.30.0",
     "@types/node": "^22.0.0",
     "@types/react": "^19.0.0",
     "@types/react-dom": "^19.0.0",
     "eslint": "^9.0.0",
     "eslint-config-next": "15.3.2",
+    "netlify-cli": "^21.2.0",
     "postcss": "^8",
     "tailwindcss": "^4.0.0",
     "typescript": "^5"
-  },
-  "packageManager": "pnpm@8.15.6+sha256.01c01eeb990e379b31ef19c03e9d06a14afa5250b82e81303f88721c99ff2e6f"
```

Comment:
Removed the package manager statement which forced usage of pnpm - despite the README using npm